.github/workflows/peerpod-ctrl_image.yaml

```diff
@@ -0,0 +1,45 @@
+# Copyright Confidential Containers Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# Push peerpod-ctrl image
+---
+name: peerpod-ctrl image push
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'peerpod-ctrl/**'
+
+jobs:
+  peerpod_push:
+    path
+    name: Push peerpod-ctrl image
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: peerpod-ctrl
+
+    steps:
+    - name: Checkout Code
+      uses: actions/checkout@v3
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v2
+    - name: Login to Quay container Registry
+      uses: docker/login-action@v2
+      with:
+        registry: quay.io
+        username: ${{ secrets.QUAY_USERNAME }}
+        password: ${{ secrets.QUAY_PASSWORD }}
+    - name: Build and push
+      uses: docker/build-push-action@v3
+      with:
+        tags: |
+          quay.io/confidential-containers/peerpod-ctrl:latest
```

Comment:
Can we do a dual push like we do for CAA? Where we push latest and one with the commit as a tag?